tests/unit/lms/resources/_js_config_test.py

```diff
@@ -47,18 +47,30 @@ def test_it(
         )
         assert authToken == bearer_token_schema.authorization_param.return_value
 
-    def test_it_is_None_for_non_lti_users(
-        self, authToken, BearerTokenSchema, context, pyramid_request
-    ):
-        pyramid_request.lti_user = None
-
-        assert JSConfig(context, pyramid_request).config["authToken"] is None
+    @pytest.mark.usefixtures("no_lti_user")
+    def test_it_is_None_for_non_lti_users(self, authToken):
+        assert authToken is None
```

Comment:
The addition of the `no_lti_user` fixture enables tests like this one, which require the `pyramid_request.lti_user` to be set to `None` _before_ the `JSConfig` object is initialized, to use fixtures like the `authToken` fixture which initialize the `JSConfig` for the test.

Move more of the test set up code out of the tests themselves and into the fixtures. Makes that setup code reusable as fixtures. Makes the test methods themselves simpler.